examples/libuv/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017, Aaditya Kalsi
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-libuv)
+
+# download libuv
+hunter_add_package(libuv)
+
+if(NOT EXISTS "${libuv_LICENSES}")
+  message(FATAL_ERROR "File not found: ${libuv_LICENSES}")
+endif()
+
+message("License: ${libuv_LICENSES}")
+message("CMAKE_PREFIX_PATH: ${CMAKE_PREFIX_PATH}")
+
+find_package(libuv REQUIRED)
+find_package(Threads)
+add_executable(example example.c)
+
+target_link_libraries(example uv ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
If `Threads` is dependency of `libuv` then it shold be set in `*Config.cmake` instead of user's code.